src/sql/workbench/services/accountManagement/browser/accountPickerImpl.ts

```diff
@@ -193,6 +252,32 @@ export class AccountPicker extends Disposable {
 		return null;
 	}
 
+	private renderTenantLabel(container: HTMLElement): IDisposable | null {
+		if (container.hasChildNodes()) {
+			for (let i = 0; i < container.childNodes.length; i++) {
+				container.removeChild(container.childNodes.item(i));
+			}
+		}
+
+		const selectedTenants = this._tenantList.getSelectedElements();
+		const tenant = selectedTenants ? selectedTenants[0] : undefined;
+		if (tenant) {
+			const row = DOM.append(container, DOM.$('div.selected-tenant-container'));
+			const label = DOM.append(row, DOM.$('div.label'));
+
+			// TODO: Pick between the light and dark logo
+			label.innerText = tenant.displayName;
+		}
+		return null;
```

Comment:
undefined

Reply:
Copy pasted from renderAccountLabel, should Is till change it?

Reply:
Might as well if we're doing this kind of refactor anyways IMO